package.json

```
@@ -53,11 +53,11 @@
     "TypeScript"
   ],
   "dependencies": {
-    "@typescript-eslint/parser": "^5.52.0",
+    "@typescript-eslint/parser": "^6.4.0",
```

```suggestion
    "@typescript-eslint/parser": "^6.0.0",
```
It's the minimal version compatible with the plugin announced to the package manager

---

The downgrade to the 6.0.0 minimum make the tests fail because of a bug in @typescript-eslint ... 

I think it's safe to require 6.4.0 as a minimum version bug free with that module.

---

Which bug, please?

---

With the rules handling comma, StandardJS style was not properly followed.